example/app.js

```diff
@@ -179,7 +192,7 @@ class App extends React.Component {
   render() {
     return (
 
-      <section className='app'>
+      <section className={'app' + (this.state.highlightActiveBullet ? ' highlight-active-bullet' : '')}>
```

Comment:
Id like to remove the bullet customization from the example gallery, and allow users to see the extra customizations in the readme.